client-react/src/pages/app/functions/function/function-log/FunctionLogAppInsightsDataLoader.tsx

```diff
@@ -273,10 +281,10 @@ const FunctionLogAppInsightsDataLoader: React.FC<FunctionLogAppInsightsDataLoade
   }, []);
 
   useEffect(() => {
-    setShowFilteredLogsMessage(getShowFilteredLogsMessage());
+    setShowFilteredLogsMessage(useNewFunctionLogsApi() && !!liveLogsSessionId);
```

Comment:
If using `useMemo` instead of `useCallback`:
```suggestion
    setShowFilteredLogsMessage(useNewFunctionLogsApi && !!liveLogsSessionId);
```